flink-core/src/main/java/org/apache/flink/configuration/ResourceManagerOptions.java

```diff
@@ -33,6 +33,45 @@
 		.key("resourcemanager.job.timeout")
 		.defaultValue("5 minutes");
 
+	public static final ConfigOption<Integer> LOCAL_NUMBER_RESOURCE_MANAGER = ConfigOptions
+		.key("local.number-resourcemanager")
+		.defaultValue(1);
+
+	public static final ConfigOption<Integer> RESOURCE_MANAGER_IPC_PORT = ConfigOptions
+		.key("resourcemanager.rpc.port")
+		.defaultValue(0);
+
+	/**
+	 * Percentage of heap space to remove from containers (YARN / Mesos), to compensate
+	 * for other JVM memory usage.
+	 */
+	public static final ConfigOption<Float> CONTAINERIZED_HEAP_CUTOFF_RATIO = ConfigOptions
+		.key("containerized.heap-cutoff-ratio")
+		.defaultValue(0.25f)
+		.withDeprecatedKeys(ConfigConstants.YARN_HEAP_CUTOFF_RATIO);
```

Comment:
let's use the actual key here. I'd like to end up in a state where we could remove the deprecated parts of the ConfigConstants class without requiring other changes.